.github/workflows/deploy.yml

```diff
@@ -0,0 +1,59 @@
+name: Deploy to fly.io
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-frontend:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '17'
```

Comment:
i think the node version is at least 18+. not sure if 17 got all the features. try 18 to be safe